cluster/v1beta2/0000_01_clusters.open-cluster-management.io_managedclustersetbindings.crd.yaml

```diff
@@ -19,11 +19,11 @@ spec:
     schema:
       openAPIV3Schema:
         description: ManagedClusterSetBinding projects a ManagedClusterSet into a
```

Comment:
Suggestion: ManagedClusterSetBinding projects a ManagedClusterSet into a
          certain namespace. You can create a ManagedClusterSetBinding in
          a namespace and bind it to a ManagedClusterSet if both have a RBAC rules
          to CREATE on the virtual subresource of managedclustersets/bind. Workloads
          that you create in the same namespace can only be distributed to ManagedClusters
          in ManagedClusterSets that are bound in this namespace by higher-level controllers.

Reply:
Updated.